scripts/populateDB.sh

```diff
@@ -0,0 +1,280 @@
+#!/usr/bin/env bash
+
+export HOST=http://localhost:8080
```

Comment:
Oops... this needs to be `http://localhost:8080/v1.0` after https://github.com/mozilla-sensorweb/sensorthings/commit/169076952faa929aaad1ce8509cf146ddac50904 landed